src/lib/Components/FilterModal/FilterModal.tsx

```diff
@@ -214,6 +214,8 @@ export const FilterOptions: React.FC<FilterOptionsProps> = (props) => {
 
   const concreteAggregations = state.aggregations ?? []
 
+  const isClearAllButtonEnabled = state.appliedFilters.length > 0 || state.appliedFilters.length > 0
```

Comment:
```suggestion
  const isClearAllButtonEnabled = state.appliedFilters.length > 0 || state.selectedFilters.length > 0
```

Based on the conversation, I _think_ this is what you had in mind here. Right now it's checking the same value twice.

Reply:
oops thanks!